classes/01class/exercises/c01-aws02/afaryy/README.md

```diff
@@ -0,0 +1,153 @@
+# C01-AWS02
+## S3 (C01-aws02)
+- Use the CLI to create a S3 bucket and upload a file to it.
+- Access the file on the S3 bucket from any the EC2 instance created in the previous exercise.
+- Submit a PR to the DevOpsAcademy repo under the `/classes/01class/exercises/c01-aws02/<github-username>` folder of this class with a brief description of what you did to achieve this objective. The file containing the solution you used must be named README.md
+
+
+## Commands Execution Output
+
+- Commands to create the S3 Bucket and to copy the file to the bucket:
+```
+aws s3 mb s3://da-co1aws02bucket
+--Output:
+make_bucket: da-co1aws02bucket
+
+aws s3 cp sample.txt s3://da-co1aws02bucket
+--Output:
+upload: .\sample.txt to s3://da-co1aws02bucket/sample.txt
+
+aws s3 ls s3://da-co1aws02bucket
+--Output:
+2020-07-05 12:58:19         19 sample.txt
+```
+- Commands to allow the EC2 instance to access the files in S3:
+ 1. Create a trust policy:
+```JSON
+$ cat da-c01aws02-EC2TrustPolicy.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+```
+
+ 2. Create a S3 access policy(da-c01aw02-S3ReadPolicy.json):
+```JSON
+$ cat da-c01aw02-S3ReadPolicy.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "dac01aw02S31",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:GetBucketLocation"
```

Comment:
`GetBucketLocation` is not needed for this exercise

Reply:
Thank you. I have corrected it.